### ISSUE TITLE
Combobox does not render accessibility required attributes

### ISSUE THREAD

**Issue by ylux587** [telerik/kendo-ui-core#5909]
### Bug report

Combobox does not render accessibility required attributes

### Reproduction of the problem
Render a combox and observe the generated html

### Current behavior

-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Form element does not have an implicit (wrapped) <label>
-   Form element does not have an explicit <label>
-   Element has no title attribute or the title attribute is empty

### Expected/desired behavior
- Should have non empty aria-label and aria-labelledby attributes
- Form element should have an implicit or explicit <label> element
- Element should have a non-empty title attribute

### Environment

* **Kendo UI version:** Kendo UI version: 2020.2.513
* **jQuery version:** 3.5.1
* **Browser:**Chromium Edge Version: 84.0.488.1 (Official build) dev (64-bit)

**Comment by GaloisGirl** [comment]
The programmer should add the aria label, as seen here: https://demos.telerik.com/kendo-ui/combobox/index

The Combobox itself doesn't know what its label should be.

**Comment by dmanova** [comment]
Consider the issue as answered with the previous comment and closing it.
